packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -862,7 +862,7 @@ module.exports = function(log, error) {
   // Update : accounts, emails
   // Set    : emailVerified = true if email is in accounts table or isVerified = true if on email table
   // Where  : uid = $1, emailCode = $2
-  var VERIFY_EMAIL = 'CALL verifyEmail_8(?, ?)';
+  var VERIFY_EMAIL = 'CALL verifyEmail_9(?, ?)';
```

Comment:
We will need to add a few asserts for this, the spots where we return the email are in the `accountEmails`  and `getSecondaryEmail` procedure. I would find relevant test cases that use these procedures and assert that they return the expected `verifiedAt` values.